src/helpers/puppeteerDriver.js

```diff
@@ -0,0 +1,51 @@
+import puppeteer from 'puppeteer';
+import Events from 'events';
+
+class Driver {
+
+	/**
+	 * Constructs a Driver instance
+	 * @return {object}
+	 */
+	constructor() {
+		this.emitter = new Events.EventEmitter();
+		this.browser = null;
+	}
+
+	/**
```

Comment:
Small thing, but it looks like there are mixed spaces and tabs in this file, can we convert to tabs?